dependencies.yaml

```diff
@@ -258,10 +258,10 @@ dependencies:
       - output_types: [conda, requirements, pyproject]
         packages:
           - click
-          - lazy_loader>=0.1
-          - numpy>=1.23.4,<3.0a0
-          - scikit-image>=0.19.0,<0.25.0a0
-          - scipy>=1.6.0
+          - lazy-loader>=0.4
+          - numpy>=1.24,<3.0a0
```

Comment:
Is there a reason for the new lower bound on `numpy`? We have `1.23.4` as the lower bound across all of RAPIDS right now. If things are still compatible with 1.23.4, we should continue using that for now.

Reply:
I see scikit-image updated their pinning upstream. This is fine, it can be left as-is.

Reply:
Let me put is back to 1.23.4 in that case. It is only for the most recent release of scikit-image that a newer version would need to be pulled in. Older numpy should work for scikit-image<=0.24